code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -43,6 +43,8 @@
 			//remove just_spawned protection if no longer processing this cell
 		SSair.add_to_active(src, 0)
 	else
+		if(heatless == 1)
```

Comment:
No it purposefully got changed to only heat up the air if it cannot cause a fire directly.

Reply:
That just doesn't make much sense to me, but ok.